corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -73,14 +76,24 @@ def iter_all_rows(reindex_accessor):
     reindex accessor
     """
     for db_alias in reindex_accessor.sql_db_aliases:
-        docs = reindex_accessor.get_docs(db_alias, reindex_accessor.startkey_min_value)
+        docs = reindex_accessor.get_docs(db_alias)
         while docs:
             for doc in docs:
                 yield doc
 
-            start_from_for_db = getattr(doc, reindex_accessor.startkey_attribute_name)
-            last_id = doc.id
-            docs = reindex_accessor.get_docs(db_alias, start_from_for_db, last_doc_pk=last_id)
+            last_id = getattr(doc, reindex_accessor.primary_key_field_name)
+            docs = reindex_accessor.get_docs(db_alias, last_doc_pk=last_id)
+
+
+def iter_all_ids(reindex_accessor):
+    for db_alias in reindex_accessor.sql_db_aliases:
+        docs = reindex_accessor.get_doc_ids(db_alias)
```

Comment:
Should this be `get_docs`?  Looks like you're using both get_doc_ids nad get_docs here

Reply:
the other one should have been `get_doc_ids`